### PR TITLE
feat: addition of taxi limitations (WIP) to ch4 of BG

### PR DESCRIPTION
**Pull request by Popespice**
## Summary
Added taxi limitations section to ch 4 of BG. Section is WIP.

### Location
docs/pilots-corner/a380x/a380x-beginner-guide/04_engine-start-taxi.md

Discord username (if different from GitHub): Popespice

### CODE REVIEW

File: docs/pilots-corner/a380x/a380x-beginner-guide/04_engine-start-taxi.md [hunk 1]
```diff
@@ -306,6 +303,27 @@ let the aircraft accelerate again. It is recommended to be between 8 and 10 knot
 If it is required to perform a sharp turn immediately, we may need a bit more thrust. Try not to perform the brake 
 check while in a turn, as we don't want to come to a complete stop while turning.
 
+### Taxi Limitations
+
+It is generally recommended that pilots only taxi on taxiways large enough to support the aircraft. Because of how wide the wingspan of the A380 is, we need large taxiways available to use. Appropriate taxiways are commonly found at large, international airports, such as EGLL, EDDM, KJFK, CYYV, etc.
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-beginner-guide/04_engine-start-taxi.md:308]: I would add that usually airports have special taxi charts for the A380 to show either allowed or forbidding taxi ways. 
Example image can be done using either EDDM or EDDF

image

File: docs/pilots-corner/a380x/a380x-beginner-guide/04_engine-start-taxi.md [hunk 2]
```diff
@@ -5,13 +5,10 @@
 This guide will explain the correct procedures to accomplish a pushback with engine start and perform a safe taxi to the departure runway.
 
 !!! warning "Disclaimer"
-    The level of detail in this guide is meant to get a FlyByWire A320neo beginner safely from the terminal to the runway hold short point.
+    The level of detail in this guide is meant to get a FlyByWire A380X beginner safely from the terminal to the runway hold short point.
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-beginner-guide/04_engine-start-taxi.md]:
```suggestion
    The level of detail in this guide is meant to get an Airbus A380 beginner safely from the terminal to the runway hold short point.
```

File: docs/pilots-corner/a380x/a380x-beginner-guide/04_engine-start-taxi.md [hunk 3]
```diff
@@ -306,6 +303,27 @@ let the aircraft accelerate again. It is recommended to be between 8 and 10 knot
 If it is required to perform a sharp turn immediately, we may need a bit more thrust. Try not to perform the brake 
 check while in a turn, as we don't want to come to a complete stop while turning.
 
+### Taxi Limitations
+
+It is generally recommended that pilots only taxi on taxiways large enough to support the aircraft. Because of how wide the wingspan of the A380 is, we need large taxiways available to use. Appropriate taxiways are commonly found at large, international airports, such as EGLL, EDDM, KJFK, CYYV, etc.
+
+<p style="color:yellow; font-size:18px;">WIP</p>
+
+??? tip "2 engine taxi"
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-beginner-guide/04_engine-start-taxi.md:312]: Not sure this is required for the beginner guide. But we can keep it in.

Comment (Popespice) [reply docs/pilots-corner/a380x/a380x-beginner-guide/04_engine-start-taxi.md:312]: Yeah i was iffy on putting it in

**Pull request merged**